includes/settings/class-wcpdf-settings-debug.php

```diff
@@ -552,11 +552,31 @@ private function renumber_or_delete_document( $document, $delete_or_renumber ) {
 		if ( $document && $document->exists() ) {
 			switch ( $delete_or_renumber ) {
 				case 'renumber':
-					if ( is_callable( array( $document, 'init_number' ) ) ) {
-						$document->init_number();
-						$return = true;
-					} elseif ( 'packing-slip' === $document->get_type() && is_callable( array( WPO_WCPDF_Pro()->functions, 'init_packing_slip_number' ) ) ) {
-						WPO_WCPDF_Pro()->functions->init_packing_slip_number( $document );
+					// pro and third party documents
+					if ( 'invoice' !== $document->get_type() ) {
```

Comment:
Can we also remove this condition?
So the whole renumber case would be like this: if the `add_number()` is callable, call it, else if `init_number()` is callable, call it.